clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -62,6 +62,12 @@ public AbstractConfig(ConfigDef definition, Map<?, ?> originals) {
         this(definition, originals, true);
     }
 
+    public AbstractConfig(Map<String, Object> parsedConfig) {
+        this.values = parsedConfig;
+        this.originals = new HashMap<>();
```

Comment:
Should we just save the `parsedConfig` as `originals`? Not sure if that provides any value